dsalgo/kruskals.py

```diff
@@ -0,0 +1,83 @@
+# Kruskal's algorithm for Minimum Spanning Tree in Python
+
+from dataclasses import dataclass, field
```

Comment:
@Aryamanz29  Is this library required?

Reply:
@devkapilbansal  no firstly he should create graph using then define krushkal's algo() inside it. Library needed for this task: collections